src/styles/page/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  max-width: 200px;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background: $page-color;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__container {
+    width: 100%;
+  }
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color;
+    margin-bottom: 4px;
+  }
+
+  &:hover &__title {
+    color: $hover-color;
+    transition: transform 0.3s;
```

Comment:
Set transition to the selector itself instead of `hover` only so it will work in both ways